xscen/aggregate.py

```diff
@@ -697,7 +697,7 @@ def produce_horizon(
     indicators:  Union[str, Path, Sequence[Indicator], Sequence[Tuple[str, Indicator]]]
         Indicators to compute. It will be passed to the `indicators` argument of `xs.compute_indicators`.
     periods: list
-        Either [start, end] or list of [start_year, end_year] for the period(s) to be evaluated.
+        List of periods to be evaluated. Each period should be a list of [start, end] or an int/float for the warming level.
```

Comment:
so it can't be ['1991','2020'] because that would be interpreted as +1991 degC and +2020degc in warming levels ?

Reply:
Better ? However yes, `periods=[1991, 2020]` (with `int` instead of `str`) would get understood as two warming levels.
`Either ['start', 'end'], or list of periods to be evaluated. Each period should be a list of [start, end] or an int/float for the warming level.`

Reply:
yes thanks !

Reply:
In a yaml, would:
```
periods:
  - 1991
  - 2020
 ```
 be ok ? is it understood as str or int ?

Reply:
It would not. An alternative might be to forget supporting `int/float` and instead require something like `+2`?

Reply:
maybe we can have a check if it is above 1000 we can assume it is a date ?

Reply:
I feel like people will expect it to work the same as other periods and it could lead to errors...

Reply:
Alright, that last commit should cover everything.